README.md

```diff
@@ -60,6 +61,8 @@ class MicrometerLogzio {
          @Override
          public String uri() {
             return "<<listener-url>>";
+            // example:
+            // return "https://listener.logz.io:5053"; 
```

Comment:
return "https://<<listener-url>>:8053";
Also the example should be with port 8053